# A simple 2d bounding box message

float64 x_min
float64 y_min
float64 x_size
float64 y_size